bootstrap.py

```diff
@@ -407,7 +410,10 @@ def install_puppet_agent():
     print_generic("Running Puppet in noop mode to generate SSL certs")
     print_generic("Visit the UI and approve this certificate via Infrastructure->Capsules")
     print_generic("if auto-signing is disabled")
-    exec_failexit("/usr/bin/puppet agent --test --noop --tags no_such_tag --waitforcert 10")
+    if os.path.isfile('/usr/bin/puppet'):
```

Comment:
can we turn that around and test for `/opt/puppetlabs/puppet/bin/puppet`? and in the else case, run puppet without a path at all, so the shell can find it even in another folder?